packages/storybook/package.json

```diff
@@ -37,20 +35,11 @@
     "@storybook/addon-viewport": "^6.1.9",
     "@storybook/react": "^6.1.9",
     "@storybook/theming": "^6.1.9",
-    "@types/jest": "^26.0.0",
-    "@types/jest-image-snapshot": "^4.1.0",
-    "@types/puppeteer": "^5.4.0",
     "babel-loader": "^8.1.0",
     "core-js": "^3.6.5",
-    "jest": "^26.0.1",
-    "jest-image-snapshot": "^4.1.0",
-    "jest-puppeteer": "^4.4.0",
-    "puppeteer": "^4.0.0",
     "react-docgen-typescript-loader": "^3.7.2",
     "source-map-loader": "^1.0.2",
-    "start-server-and-test": "^1.11.3",
     "string-replace-loader": "^3.0.1",
-    "ts-loader": "^8.0.2",
-    "typescript": "^4.1.2"
```

Comment:
no typescript dependency?

Reply:
"typescript" is already defined in the root package.json. I think that we should aim, as much as possible, to handle a single version of the dependencies.

Reply:
yes but would that work in netlify or as an independent pkg? 🤔

Reply:
Netlify is green. And yes, the base folder is configured to be the root.